docs/source/tools/codegen.rst

```diff
@@ -83,29 +83,20 @@ Here is an example::
         package-prefix: com.daml.quickstart.iou
         output-directory: java-codegen/src/main/java
         verbosity: 2
-      scala:
-        package-prefix: com.daml.quickstart.iou
-        output-directory: scala-codegen/src/main/scala
-        verbosity: 2
 
-You can then run the above configuration to generate your **Java** or **Scala** code::
+You can then run the above configuration to generate your **Java** code::
 
-    $ daml codegen [js|java|scala]
+    $ daml codegen java
```

Comment:
again probably a silly one , should we get be keeping `js` here given the changes above ? and the line below ?

Reply:
The line above says:

> You can then run the above configuration to generate your **Java** code::

So I assumed that the inclusion of `js` was a mistake. @robin-da?

Reply:
I also think this is a mistake.